docker_push_ssh/cli.py

```diff
@@ -204,6 +204,9 @@ def main():
 
     parser.add_argument("-p", "--ssh-port", type=str, help="[optional] Port on ssh host to connect to. (Default is 22)", default="22")
 
+    parser.add_argument("-r", "--registry-port", type=str,
```

Comment:
From the look of things you are changing the port used on localhost rather than the remote host. Can you update the description below to reflect this?